dev/reference/security/role-based-authorization.md

```diff
@@ -0,0 +1,281 @@
+---
+title: 基于角色的访问控制（该功能还在测试中）
```

Comment:
```suggestion
title: 基于角色的访问控制
```
It is not appropriate to add this note in the title (makes it informal and long). We can add this note as a warning under the first heading.